test/lib/code_corps/model/project_test.exs

```diff
@@ -83,8 +83,17 @@ defmodule CodeCorps.ProjectTest do
 
   describe "create_changeset/3" do
     test "with valid attributes" do
+      category = insert(:category)
+      skill = insert(:skill)
       organization = insert(:organization)
-      attrs = %{title: "A title", organization_id: organization.id}
+      attrs = %{
+        "categories_ids" => [category.id],
+        "cloudinary_public_id" => "foo123",
+        "description" => "Description",
+        "skills_ids" => [skill.id],
```

Comment:
We are using `coalesce_id_string` above. Wouldn't that mean we're supposed to be sending csv ids here, instead of a list?

Reply:
No, because this is how JaSerializer modifies the data.